event_listener/requirements.txt

```diff
@@ -5,3 +5,5 @@ django-model-utils==3.1.2
 ethereum==2.3.2
 pymongo==3.7.2
 web3==4.8.2
```

Comment:
Are we planning to include pylint inthe requirements?

If so, I believe it would be
```
pylint==2.1.1
```

At least this is what I have locally for my python3.7